packages/deployer/subtasks/load-deployment.js

```diff
@@ -25,6 +25,10 @@ subtask(SUBTASK_LOAD_DEPLOYMENT, 'Loads deployment artifacts for a particular in
     const previousDeploymentFile =
       deploymentFiles.length > 1 ? deploymentFiles[deploymentFiles.length - 2] : null;
 
+    if (!currentDeploymentFile) {
+      throw new Error(`Could not find deployment files with ${JSON.stringify(info)}`);
+    }
```

Comment:
Just added a more clear error when missing a deployment file, this would have saved me some time.